pages/linux/ip-route.md

```diff
@@ -0,0 +1,32 @@
+# ip route
+
+> IP Routing table management subcommand.
+> More information: <https://manned.org/ip-route>.
+
+- Display the routing table:
+
+`ip route {{show|list}}`
+
+- Add a default route using gateway forwarding:
+
+`sudo ip route add default via {{gateway_ip}}`
+
+- Add a default route using `eth0`:
+
+`sudo ip route add default dev {{eth0}}`
+
+- Add a static route:
+
+`sudo ip route add {{destination_network_ip}} via {{gateway_ip}} dev {{eth0}}`
```

Comment:
Why not just `destination_ip`?
```suggestion
`sudo ip route add {{destination_ip}} via {{gateway_ip}} dev {{eth0}}`
```

Reply:
ok, no problemo for me. In fact, you're right, you can definitely add a route to a specific destination host.